{
  "uid" : "557fdfda9bc61353",
  "name" : "test_add1[文-9.3-参数大小超出范围]",
  "fullName" : "test.cases.add.test_add.TestAdd#test_add1",
  "historyId" : "6eedc70cee67ca61079c3fde33dedfba",
  "time" : {
    "start" : 1660620753118,
    "stop" : 1660620753118,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "TypeError: '>' not supported between instances of 'str' and 'int'",
  "statusTrace" : "self = <test.cases.add.test_add.TestAdd object at 0x000002A6A29159B0>, x = '文', y = 9.3\nz = '参数大小超出范围'\n\n    @pytest.mark.add\n    @pytest.mark.parametrize('x,y,z',yaml.safe_load(open(my_dir+\"/adddata.yaml\")))\n    def test_add1(self,x,y,z):\n        # with allure.step():\n        self.cal=Calculator()\n>       result=self.cal.add(x,y)\n\ntest\\cases\\add\\test_add.py:18: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <script.calculator.Calculator object at 0x000002A6A2954CC0>, a = '文', b = 9.3\n\n    def add(self, a, b):\n    \n>       if a > 99 or a < -99 or b > 99 or b < -99:\nE       TypeError: '>' not supported between instances of 'str' and 'int'\n\nscript\\calculator.py:4: TypeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_xunit_setup_method_fixture_TestAdd",
    "time" : {
      "start" : 1660620753117,
      "stop" : 1660620753117,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "_xunit_setup_class_fixture_TestAdd",
    "time" : {
      "start" : 1660620753061,
      "stop" : 1660620753062,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "TypeError: '>' not supported between instances of 'str' and 'int'",
    "statusTrace" : "self = <test.cases.add.test_add.TestAdd object at 0x000002A6A29159B0>, x = '文', y = 9.3\nz = '参数大小超出范围'\n\n    @pytest.mark.add\n    @pytest.mark.parametrize('x,y,z',yaml.safe_load(open(my_dir+\"/adddata.yaml\")))\n    def test_add1(self,x,y,z):\n        # with allure.step():\n        self.cal=Calculator()\n>       result=self.cal.add(x,y)\n\ntest\\cases\\add\\test_add.py:18: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <script.calculator.Calculator object at 0x000002A6A2954CC0>, a = '文', b = 9.3\n\n    def add(self, a, b):\n    \n>       if a > 99 or a < -99 or b > 99 or b < -99:\nE       TypeError: '>' not supported between instances of 'str' and 'int'\n\nscript\\calculator.py:4: TypeError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "4fb7da225ba9699b",
      "name" : "stdout",
      "source" : "4fb7da225ba9699b.txt",
      "type" : "text/plain",
      "size" : 26
    } ],
    "parameters" : [ ],
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 1,
    "hasContent" : true,
    "stepsCount" : 0
  },
  "afterStages" : [ {
    "name" : "_xunit_setup_method_fixture_TestAdd::0",
    "time" : {
      "start" : 1660620753134,
      "stop" : 1660620753134,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  }, {
    "name" : "_xunit_setup_class_fixture_TestAdd::0",
    "time" : {
      "start" : 1660620753161,
      "stop" : 1660620753161,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0,
    "hasContent" : false,
    "stepsCount" : 0
  } ],
  "labels" : [ {
    "name" : "tag",
    "value" : "add"
  }, {
    "name" : "parentSuite",
    "value" : "test.cases.add"
  }, {
    "name" : "suite",
    "value" : "test_add"
  }, {
    "name" : "subSuite",
    "value" : "TestAdd"
  }, {
    "name" : "host",
    "value" : "LAPTOP-BEO6NH7O"
  }, {
    "name" : "thread",
    "value" : "11036-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.cases.add.test_add"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "x",
    "value" : "'文'"
  }, {
    "name" : "y",
    "value" : "9.3"
  }, {
    "name" : "z",
    "value" : "'参数大小超出范围'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "add" ]
  },
  "source" : "557fdfda9bc61353.json",
  "parameterValues" : [ "'文'", "9.3", "'参数大小超出范围'" ]
}